larray_editor/arraywidget.py

```diff
@@ -760,26 +752,70 @@ def dropEvent(self, event):
         else:
             event.ignore()
 
-    def set_data(self, data, bg_value=None):
-        # TODO: in the future, data should either be an adapter directly or we should instantiate one here depending
-        # on the type of data it received. Having a single adapter instance and using set_data on it like we do now
-        # cannot work because we will need a different adapter class for different data types.
-        data = la.aslarray(data)
-
-        axes = data.axes
-        display_names = axes.display_names
+    def _reset_minmax(self):
+        self.model_data.reset_minmax()
+
+    def _update_models(self, reset_model, reset_minmax):
```

Comment:
reset_model is ambiguous. I thought (before looking at the code) that it caused all models to be reset. rename it to reset_model_data or something like that?

Reply:
Done